### PR TITLE
Network EGP data only to connected players

### PR DESCRIPTION
**Pull request by wrefgtzweve**
This change makes EGPs only network to players that are connected to the EGP, should reduce network usage massively and especially help larger servers.

It seems to work in my tests but i haven't been able to throughly make sure nothing is backwards broken. Making this a draft for now for feedback.

### CODE REVIEW

File: lua/entities/gmod_wire_egp/lib/egplib/transmitreceive.lua [hunk 1]
```diff
@@ -574,7 +574,7 @@ if (SERVER) then
 		end
 	end)
 
-	function EGP:SendDataStream( ply, entid )
+	function EGP:SendDataStream( ply, entid, silent )
```
Comment (DerelictDrone) [inline lua/entities/gmod_wire_egp/lib/egplib/transmitreceive.lua:577]: Is this used?

Comment (wrefgtzweve) [reply lua/entities/gmod_wire_egp/lib/egplib/transmitreceive.lua:577]: Yeah see line 625

**Comment by wrefgtzweve** [comment]
I've been using this on a server for a while now and i havent seen any complaints regarding EGP.

**Comment by brandonsturgeon** [comment]
> I've been using this on a server for a while now and i havent seen any complaints regarding EGP.

Can corroborate, seems to work as expected 👍 

Good change!

**Pull request merged**